src/pages/Review/StarRating.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import Star from './Star';
+import './Review.scss';
+
+class StarRating extends Component {
+  render() {
+    const { star } = this.props;
+    return (
+      <>
+        {[...Array(5).keys()].map(el => {
+          return star > el ? (
+            <Star active={true} key={el} />
+          ) : (
+            <Star active={false} key={el} />
```

Comment:
```suggestion
        {[...Array(5).keys()].map(num => <Star active={star > el} key={num} />
```
- el 같은 변수명은 좋지 않은 습관이니까 꼭 고쳐주시고
- t/f 값만 사용할거라면 위와 같이 수정해볼 수 있을거 같습니다

Reply:
넵!!! 수정완료 했습니다!